bedrock-entry.sh

```diff
@@ -79,7 +79,7 @@ if [ ! -f "bedrock_server-${VERSION}" ]; then
   done
 
   # ... overwrite all game files, except config files
-  unzip -q ${TMP_ZIP} -x $(ls server.properties whitelist.json permissions.json 2> /dev/null)
+  unzip -q ${TMP_ZIP} -x "$(ls server.properties whitelist.json permissions.json 2> /dev/null)"
```

Comment:
I wonder if this change broke the test

Reply:
possibly! my IDE suggested that... shouldn’t have touched :-)